frontend/app/components/comment-form/__subscribe-by-rss/comment-form__subscribe-by-rss.test.tsx

```diff
@@ -31,9 +31,9 @@ describe('<SubscribeByRSS/>', () => {
     expect(wrapper.find('.comment-form__rss-dropdown__link')).toHaveLength(3);
   });
 
-  it('should have userId in site link', () => {
-    expect(wrapper.find('.comment-form__rss-dropdown__link').at(1).prop('href')).toEqual(
-      createSubscribeUrl('site', '&user=user-1')
+  it('should have userId in Replies link', () => {
+    expect(wrapper.find('.comment-form__rss-dropdown__link').at(2).prop('href')).toEqual(
```

Comment:
I know it was here. But the right way to compare primitive values over `toBe`. `toEqual` was added for compare objects or arrays. Let's fix this ;)
```suggestion
    expect(wrapper.find('.comment-form__rss-dropdown__link').at(2).prop('href')).toBe(
```